src/main/java/thePackmaster/cards/farmerpack/Fertilizer.java

```diff
@@ -7,7 +7,8 @@
 
 
 import static thePackmaster.SpireAnniversary5Mod.makeID;
-
+@NoPools
+@NoCompendium

```

Comment:
Since this card is Special rarity, I don't think you need these annotations -- Special rarity cards won't be randomly generated (unless the code doing the random generation is bugged). I'm also not sure they do anything for colorless cards like this, they're more for cards in a characters color that shouldn't show up for some reason. I'd suggest just removing them.

Also, I think the imports needed for these to work are missing -- does this compile?